### PR TITLE
create method getFrentes

### PR DESCRIPTION
**Pull request by LeonardoCamargo31**
Create method getFrentes.

Usage:
```js
await CamaraAPI.getFrentes({idLegislatura:'56'})
await CamaraAPI.getFrentes()
```

issue: https://github.com/AcompanhaLegis/sdk-dados-abertos-camara-js/issues/5

### CODE REVIEW

File: modules/frentes.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    pagina: 1,
+};
+
+/**
+ * Wraps frentes endpoint.
+ *
+ * @param {Object} options - Options to be sent in the request.
+ * @param {Boolean} fullResponse - If true it will retrieve the whole response object, otherwise it will return only the data object inside the response.
+ */
+export const getFrentes= async (
```
Comment (kimuraz) [inline modules/frentes.js:16]: Nice job here!

I just miss the `getFrente`  which wraps `/frentes/{id}` and the ` getFrenteMembros` (`/frentes/{id}/membros`), so to fulfill the issue, can you add those here?

File: sample.js [hunk 2]
```diff
@@ -15,13 +15,15 @@ const main = async () => {
...
   const orgaosDeputado = await CamaraAPI.getOrgaosByDeputado(deputadoId);
+  const frentesParlamentar = await CamaraAPI.getFrentes()
 
   console.log('deputados', deputados);
   console.log('deputado', deputado);
   console.log('despesas', despesas);
   console.log('discursos', discursos);
   console.log('eventos deputado', eventosDeputado);
   console.log('frentes', frentes);
+  console.log('frentesParlamentar', frentesParlamentar);
```
Comment (kimuraz) [inline sample.js:26]: :+1:

**Review by kimuraz — CHANGES_REQUESTED** [review]
:1st_place_medal: Nice job! Please, check my comments, almost there! :+1:

File: modules/frentes.js (hunk 1 above)
Comment (LeonardoCamargo31) [reply modules/frentes.js:16]: okay, I will create these methods ;)

**Review by kimuraz — APPROVED** [review]
nice job! 🚀

**Pull request merged**